src/styles/_catalog.scss

```
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

Would be better if you create variable for this
```suggestion
  grid-template-columns: repeat(1, $cardWidth);
```